mapic/mistapiconnector_app.go

```diff
@@ -506,18 +525,66 @@ func (mc *mac) reconcileLoop(ctx context.Context) {
 		case <-ctx.Done():
 			return
 		case <-ticker.C:
-		case <-mc.multistreamUpdated:
+		case <-mc.streamUpdated:
 		}
 		mistState, err := mc.mist.GetState()
 		if err != nil {
 			glog.Errorf("error executing query on Mist, cannot reconcile err=%v", err)
 			continue
 		}
+		mc.reconcileStreams(mistState)
 		mc.reconcileMultistream(mistState)
 		mc.processStats(mistState)
 	}
 }
 
+func (mc *mac) reconcileStreams(mistState clients.MistState) {
```

Comment:
I wonder if it's possible to maybe add unit tests for this? Wdyt?

Reply:
Yeah ideally we would, but I won't have bandwidth for that this week since we're not officially prioritizing this as a task/project. Since this is more of a back-office flow I considered it less of a requirement for now and am willing to merge it as is. Otherwise, if no one is able to pick it up I can come back to this when back from PTO in november.

Reply:
I see. I just checked and adding the unit tests was not so difficult, so added them in 3416b6a9b6d3da4451688367a59ee3d46d0dd9e3

Reply:
Awesome, thanks!